net/isc-dhcp/files/dhcpd.init

```diff
@@ -11,8 +11,7 @@ PREFIX="update add"
 lease_file=/tmp/dhcpd.leases
 config_file=/tmp/run/dhcpd.conf
 
-dyndir=/tmp/bind
```

Comment:
Leave this line as is for now, but put it into the Bind PR as a 2nd commit so it all happens atomically.